.github/workflows/run-test-cases.yml

```diff
@@ -98,7 +98,7 @@ jobs:
   test-cases:
     needs: build-containers
     runs-on: ubuntu-18.04
-    timeout-minutes: 35
+    timeout-minutes: 45
```

Comment:
Do we want to increase that generously to avoid having to increase it later again?

Reply:
maybe 60 mins?

Reply:
makes sense. bumped it to 60